azure-pipelines.yml

```diff
@@ -100,24 +94,85 @@ jobs:
           --configuration $(_BuildConfig)
           --prepareMachine
         displayName: Unix Build / Publish
-- template: /eng/validation/templates/testing/helix.yml
```

Comment:
"_PublishType"  parameter isn't used anywhere (same reference where I'm removing this from Arcade as the above comment).